Detectors/ITSMFT/MFT/tracking/src/Tracker.cxx

```diff
@@ -113,6 +115,8 @@ void Tracker<T>::initializeFinder()
     return;
   }
 
+  std::lock_guard<std::mutex> guard(o2::mft::TrackerConfig::sTCMutex);
```

Comment:
The lock will prevent executing the code below at the same time for different tracker copes (one will wait for other), but as it is now, each copy may still assign to mBins. To avoid this, you can check right after the guard if the the mBins is already created and proceed only if it is not. In this logic, you should move the `initContainers` here:

```
std::lock_guard<std::mutex> guard(o2::mft::TrackerConfig::sTCMutex);
if (mBins) {
  return;
}
initBinContainers();
...

```

Reply:
Ok thanks to clarify, I will change it accordingly to this